src/nvme/linux.h

```diff
@@ -205,6 +205,42 @@ int nvme_gen_dhchap_key(char *hostnqn, enum nvme_hmac_alg hmac,
  */
 long nvme_lookup_keyring(const char *keyring);
 
+/**
+ * nvme_describe_key_serial() - Return key description
+ * @key_id:    Key serial number
+ *
+ * Fetches the description of the key or keyring identified
+ * by the serial number @key_id.
+ *
+ * Return: The description of @key_id or NULL on failure.
+ * The returned string needs to be freed by the caller.
+ */
+char *nvme_describe_key_serial(long key_id);
```

Comment:
It seems that the `keyctl` library sets `errno` on error so we should just forward it and document it here (and the following functions).

Reply:
Sure.

Reply:
Done with the update.